packages/block-library/src/reset.scss

```diff
@@ -17,6 +15,14 @@
 	line-height: initial;
 	color: initial;
 
+	.block-editor-block-list__layout.is-root-container {
```

Comment:
This reset file is only meant  to reset the wp-admin styles for the editor canvas to make them match the frontend. It shouldn't contain editor specific styles like this.